libs/designsystem/src/lib/components/empty-state/empty-state.component.spec.ts

```diff
@@ -1,8 +1,9 @@
+import { createHostFactory, SpectatorHost } from '@ngneat/spectator';
 import { MockComponent } from 'ng-mocks';
-import { SpectatorHost, createHostFactory } from '@ngneat/spectator';
 
-import { DesignTokenHelper } from './../../helpers/design-token-helper';
 import { IconComponent } from '../icon/icon.component';
+
+import { DesignTokenHelper } from './../../helpers/design-token-helper';
```

Comment:
I've attempted to revert this file - but whenever I commit the imports are re-ordered automatically.

Reply:
It managed to make it work!